### PR TITLE
Define abandonment

### LINKED ISSUE #2671: Clarify "abandon" in the context of version negotiation
The draft currently reads:
"When a client receives a Version Negotiation packet, it MUST abandon the current connection attempt."
However, it does not define what it is to abandon a connection attempt. Is it an abrupt termination without writing a connection close? That probably makes the most sense since the server is meant to be stateless at this stage.

### PR DESCRIPTION
**Pull request by martinthomson**
Just in case it wasn't perfectly clear from the choice of words.

Closes #2671.

### CODE REVIEW

**Comment by martinthomson** [comment]
I know how you feel.  We'll be defining "the" eventually.  I don't know where the line is, but I think that it helps in this case to be doubly explicit.

**Comment by marten-seemann** [comment]
I'm wondering how one could possibly misinterpret "abandoning" here. Furthermore, I don't think it's relevant for the server here if the client actually discarded the connection state, as long as it doesn't any more packets.

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -1130,6 +1130,8 @@ This system allows a server to process packets with unsupported versions without
 retaining state.  Though either the Initial packet or the Version Negotiation
 packet that is sent in response could be lost, the client will send new packets
 until it successfully receives a response or it abandons the connection attempt.
+As a result, the client discards all state for the connection and does not send
+any more packets.
```
Comment (janaiyengar) [inline draft-ietf-quic-transport.md]:
```suggestion
any more packets on the connection.
```

**Comment by janaiyengar** [comment]
I don't think we need to spend a lot of energy here. I agree that this doesn't seem necessary, but I'm not going to fight it either.

**Pull request merged**